packages/alpha/src/__tests__/api/monitoringTasksApi.int.spec.ts

```diff
@@ -21,14 +21,20 @@ describe('monitoring tasks api', () => {
   const monitoringTaskExternalId = `test_mt_${ts}`;
   const channelExternalId = `test_channel_mt_${ts}`;
   const sessionsApi = `/api/v1/projects/${TEST_PROJECT}/sessions`;
-  const testMtParams = {
+  const testMtModel = {
```

Comment:
```suggestion
  const testMtModel: MonitoringTaskThresholdModelCreate
```
to get a proper type inference, as you declare this outside of the request

Reply:
similar to `odel: testMtModel as MonitoringTaskThresholdModelCreate` but better